x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/hooks/use_update_tags.tsx

```diff
@@ -34,7 +34,7 @@ export const useUpdateTags = () => {
         const message =
           successMessage ??
           i18n.translate('xpack.fleet.updateAgentTags.successNotificationTitle', {
-            defaultMessage: 'Tags updated',
+            defaultMessage: 'Tag(s) updated',
```

Comment:
Fix https://github.com/elastic/kibana/issues/142182
updated toast message text to include singular and plural